phoenix-core/src/it/java/org/apache/phoenix/monitoring/BasePhoenixMetricsIT.java

```diff
@@ -102,30 +107,118 @@ void assertReadMetricsForMutatingSql(String tableName, long tableSaltBuckets,
         assertEquals("There should have been read metrics only for one table: " + tableName, 1, numTables);
     }
 
-    void assertMutationMetrics(String tableName, int numRows, Map<String, Map<MetricType, Long>> mutationMetrics) {
-        assertTrue("No mutation metrics present when there should have been", mutationMetrics.size() > 0);
+    static void assertMutationMetrics(String tableName, int numRows, boolean isUpsert,
+            Map<String, Map<MetricType, Long>> mutationMetrics) {
         for (Map.Entry<String, Map<MetricType, Long>> entry : mutationMetrics.entrySet()) {
             String t = entry.getKey();
-            assertEquals("Table name didn't match for mutation metrics", tableName, t);
+            assertEquals("Table names didn't match!", tableName, t);
             Map<MetricType, Long> p = entry.getValue();
-            assertEquals("There should have been five metrics", 5, p.size());
+            assertEquals("There should have been nine metrics", 15, p.size());
```

Comment:
There should have been `nine` metrics?  Why you are comparing 15 and p.size here?

Reply:
The new mutationQueue has 15 metrics after my change(added some metrics as part of tableMetrics), so modified to 15. modified the print statement.